pkg/controller/worker/machines.go

```diff
@@ -32,6 +32,11 @@ import (
 	awsapihelper "github.com/gardener/gardener-extension-provider-aws/pkg/apis/aws/helper"
 )
 
+const (
+	// AwsCSIDriverTopologyKey describes the topology key used by the AWS CSI driver
+	AwsCSIDriverTopologyKey = "topology.kubernetes.io/zone"
```

Comment:
Can you maybe add a comment with the issue/PR for the upstream change ? I think for the near future, there may be questions about this.

Also I would remove `Aws` prefix